include/mgp.hpp

```diff
@@ -1923,6 +1917,7 @@ void Graph::DeleteRelationship(const Relationship &relationship) { mgp::graph_de
 
 // Nodes:
 
+// TODO: remove comments if possible
```

Comment:
Reminder to not forget this

Reply:
Thanks! I’ll run unit tests and remove the comments when everything’s working.